features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScope.kt

```diff
@@ -281,6 +288,58 @@ internal class RumSessionScope(
         )
     }
 
+    private fun handleTTIDEvent(
+        event: RumRawEvent.AppStartTTIDEvent,
+        datadogContext: DatadogContext,
+        writeScope: EventWriteScope,
+        writer: DataWriter<Any>
+    ) {
+        sdkCore.newRumEventWriteOperation(datadogContext, writeScope, writer) {
+            createTTIDVitalEvent(
+                event = event,
+                datadogContext = datadogContext
+            )
+        }.submit()
+
+        rumAppStartupTelemetryReporter.reportTTID(
+            info = event.info,
+            indexInSession = appStartIndex
+        )
+
+        appStartIndex++
+    }
+
+    private fun createTTIDVitalEvent(
+        event: RumRawEvent.AppStartTTIDEvent,
+        datadogContext: DatadogContext
+    ): VitalEvent {
+        val rumContext = getRumContext()
+
+        val hasReplay = childScope?.activeView?.viewId?.let {
+            featuresContextResolver.resolveViewHasReplay(datadogContext, it)
+        }
+
+        return rumVitalEventHelper.newVitalEvent(
+            timestampMs = event.info.scenario.initialTime.timestamp + sdkCore.time.serverTimeOffsetMs,
+            datadogContext = datadogContext,
+            eventAttributes = emptyMap(),
+            customAttributes = getCustomAttributes(),
+            view = null,
+            hasReplay = hasReplay,
```

Comment:
I'm not sure if we need to set this flag at all here. Yes, there is such property, but it is only app start, so no real view yet, hence nothing to replay. We have `view = null` above anyway. WDYT?

If we don't set this property, we even simplify the code, because there is not need to pass `FeaturesContextResolver` around.